### ISSUE TITLE
How to deal with umlauts (utf8 / unicode)

### ISSUE THREAD

**Issue by x421** [snaptoken/kilo-tutorial#25]
Hey there,

first of all thank you for the great tutorial. I followed your steps and got a working editor, ( I skipped the search for now).

But I have one big problem: 
When I type umlauts, I get a spacing error. I tried to play around with the output of the step "ctrl-q" but only got weird symbols as an output. 

The problem with those chars is that they use 2 positions in the x variable of the cursor, but I fail to understand how I can determine when such a wide char is used. 
Also, maybe char is the wrong type to read that kind of input? 

Hope you can help me.

**Comment by nirs** [comment]
I don't think supporting unicode is feasible for 1k loc editor. The proper way to deal with it is to use a real editor :-)

**Comment by x421** [comment]
Even if it is out of scope for kilo, a notice in the appendix with links to learn more (after all it is a beginners tutorial) would be greate.

**Comment by yjerem** [comment]
Yes, it seems like it would be too complicated to add unicode support to the tutorial. However, you should be able to change the character encoding of your terminal to use an extended ASCII encoding (like ISO 8859-1), and then umlauts should at least show up when you type them in the editor.
